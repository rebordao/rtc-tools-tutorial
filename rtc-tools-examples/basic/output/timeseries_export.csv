time,Q_release,V_storage
2013-01-01 00:00:00,0.000000,400000.000000
2013-01-01 12:00:00,1.968390,399205.558747
2013-01-02 00:00:00,2.550627,399178.461220
2013-01-02 12:00:00,3.310417,398728.437303
2013-01-03 00:00:00,4.431306,395216.020217
2013-01-03 12:00:00,5.831536,383053.681186
2013-01-04 00:00:00,6.430396,416300.582533
2013-01-04 12:00:00,5.808507,405133.068716
2013-01-05 00:00:00,4.443870,401077.898111
2013-01-05 12:00:00,3.300260,401066.672834
2013-01-06 00:00:00,2.111729,420000.000098
2013-01-06 12:00:00,1.950000,420000.000098
