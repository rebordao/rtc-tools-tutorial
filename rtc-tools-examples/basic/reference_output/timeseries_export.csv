time,Q_release,V_storage
2013-01-01 00:00:00,0.000000,400000.000000
2013-01-01 12:00:00,2.085050,394165.839387
2013-01-02 00:00:00,2.554889,393954.635657
2013-01-02 12:00:00,3.311944,393438.648847
2013-01-03 00:00:00,4.412360,390744.692494
2013-01-03 12:00:00,5.754275,381920.000057
2013-01-04 00:00:00,6.451577,414251.881869
2013-01-04 12:00:00,5.621580,411159.617873
2013-01-05 00:00:00,4.700567,396015.121490
2013-01-05 12:00:00,3.273470,397161.230627
2013-01-06 00:00:00,2.021325,420000.004198
2013-01-06 12:00:00,1.950000,420000.004198
